src/main/kotlin/blackjack/BlackJackApplication.kt

```diff
@@ -50,8 +58,8 @@ private fun initDealer(dealerName: String): Dealer {
     return Dealer(dealerName, initDealerCard)
 }
 
-private fun initPlayers(players: List<String>): Set<Player> {
-    return players.map { Player(it, Deal.giveCards(INIT_FACE_UP)) }.toSet()
+private fun initPlayers(betInfos: List<BetInfo>): Set<Player> {
+    return betInfos.map { Player(it.name, it.betAmount, Deal.giveCards(INIT_FACE_UP)) }.toSet()
```

Comment:
[kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls](https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls)
코틀린 공식 문서에 따르면 함수 체인의 경우 아래와 같이 체인별 개행을 해주어야 합니다.
(일반적으로 체인의 첫 번째 호출에는 개행이 있어야 하지만 개행하지 않는 편이 더 합리적이라면 생략해도 괜찮습니다)

```suggestion
    return betInfos
        .map { Player(it.name, it.betAmount, Deal.giveCards(INIT_FACE_UP)) }
        .toSet()
```